package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "RandomCoffee",
+  "homepage": "http://random-coffee.fun/",
+  "version": "0.2.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.1",
+    "classnames": "^2.2.6",
+    "draft-convert": "^2.1.8",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.18.0",
+    "gh-pages": "^2.2.0",
+    "html-react-parser": "^0.7.1",
+    "js-md5": "^0.7.3",
+    "jwt-decode": "^2.2.0",
+    "leaflet": "^1.6.0",
+    "medium-draft": "^0.5.18",
+    "node-sass": "^4.13.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-calendar": "^2.19.2",
+    "react-dom": "^16.12.0",
+    "react-leaflet": "^2.6.1",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "^3.3.0",
+    "react-text-mask": "^5.4.3",
+    "react-yandex-maps": "^3.1.0",
+    "reveal": "0.0.4",
```

Comment:
Check your npm packages list. There are some unused packages, e.g. **reveal**, **draft-convert**, **axios-mock-adapter**

Reply:
@DasneiN update package dependecies and remove useless, axios-mock-adapter will be used for fetching mockable data.